soda/duckdb/tests/test_duckdb.py

```diff
@@ -21,3 +23,98 @@ def test_pandas_df(data_source_fixture: DataSourceFixture):
     scan.execute(allow_warnings_only=True)
     scan.assert_all_checks_pass()
     scan.assert_no_error_logs()
+
+
+def test_df_from_csv(data_source_fixture: DataSourceFixture, tmp_path: Path):
+    import pandas as pd
+
+    csv_folder = tmp_path / "csv"
+    csv_folder.mkdir()
+    csv_path = csv_folder / "csv_dataset.csv"
+
+    test_df = pd.DataFrame.from_dict({"i": [1, 2, 3, 4], "j": ["one", "two", "three", "four"]})
+    test_df.to_csv(csv_path)
+
+    scan = data_source_fixture.create_test_scan()
+    scan.set_data_source_name("csv_dataset")
+    scan.add_configuration_yaml_str(
+        f"""
+          data_source csv_dataset:
+            type: duckdb
+            path: {csv_path}
+        """
+    )
+    scan.add_sodacl_yaml_str(
+        f"""
```

Comment:
Can we remove the f-strings as there are no vars in the string and code analysis is complaining?

```suggestion
        """
```